__tests__/translateTrackingEventDetails.test.ts

```diff
@@ -0,0 +1,20 @@
+import translateTrackingEventDetails from '../src/translateTrackingEventDetails';
+
+it('returns undefined if details are empty', () => {
+  expect(translateTrackingEventDetails({})).toBeUndefined();
+});
+
+it('returns undefined if details only contains value', () => {
+  expect(translateTrackingEventDetails({ value: 12345 })).toBeUndefined();
+});
+
+it('returns object without value attribute', () => {
```

Comment:
nit: I'd read back through the test names as sentences to make sure they read well that way.

For example "It returns object without value attribute." vs "It returns an object without the value attribute."